ibm_mq/tests/common.py

```diff
@@ -4,14 +4,25 @@
 
 import os
 
+import pytest
+
 from datadog_checks.dev import get_docker_hostname
+from datadog_checks.dev.ci import running_on_ci
+from datadog_checks.dev.utils import ON_WINDOWS
 
 # Ignore missing library to not require it for e2e
 try:
     from datadog_checks.ibm_mq.metrics import COUNT, GAUGE
 except ImportError:
     COUNT = GAUGE = ''
 
+RUNNING_ON_WINDOWS_CI = ON_WINDOWS and running_on_ci()
+skip_windows_ci = pytest.mark.skipif(RUNNING_ON_WINDOWS_CI, reason='MQ server cannot be setup on Windows VMs in CI')
+
+# Test that we can at least import the library
+if RUNNING_ON_WINDOWS_CI:
+    import pymqi  # noqa: F401
```

Comment:
This is already covered in the unit tests that are running on windows CI

Reply:
https://github.com/DataDog/integrations-core/blob/master/ibm_mq/tests/test_ibm_mq_unit.py#L15